.gitignore

```diff
@@ -19,7 +20,11 @@ __pycache__/
 .eggs/
 .python-version
 
-# JS
-tests/node_modules
-tests/lib
-tests/elfs
+# Virtual env for sideload (macOS and Windows)
+ledger/
+
+# Temporary directory with snapshots taken during test runs
+tests/snapshots-tmp/
+
+# Temporary directory with all the plugin binairies (for each device)
+tests/lib_binaries
```

Comment:
On exchange I have created a token file inside to have github create the directory
https://github.com/LedgerHQ/app-exchange/blob/develop/test/python/lib_binaries/.compiled_sideloaded_applications_go_there

Then here
https://github.com/LedgerHQ/app-exchange/blob/develop/.gitignore#L44
we ignore all `elf` and `zip`